decentraland/unity-renderer
# feat: moving skybox to addressables

## What does this PR change?

Moves the skybox assets to addressables

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/moving-skybox-to-addressables
2. Play with the skybox. It should work as it currently does
3. Enter and exit DCL changing the skybox settings. Move from Dynamic to static and viceversa. When rentering Decentraland, the setting should be reflected as it was saved.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
